docs/functions/Personality/Get-VCFPersonality.md

```diff
@@ -0,0 +1,46 @@
+# Get-VCFPersonality
+
+### Synopsis
+Get the personalities
```

Comment:
What kind of personalities?!

Reply:
My assumption is vSphere Lifecycle Manager personalities - Let me add something